src/event_motion.rs

```diff
@@ -3,9 +3,26 @@
 // Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
 
 use glib::translate::*;
+use ffi::GdkModifierType;
 
 #[derive(Clone, Debug)]
 pub struct EventMotion(::Event);
 
 event_wrapper!(EventMotion, GdkEventMotion);
 event_subtype!(EventMotion, MotionNotify);
+
+impl EventMotion {
+    pub fn get_position(self: &Self) -> (f64, f64) {
+        let x = self.as_ref().x;
+        let y = self.as_ref().y;
+        (x, y)
+    }
+
+    pub fn get_state(self: &Self) -> GdkModifierType {
```

Comment:
Maybe I wasn't clear, I want `ModifierType` here, it's reexported in this crate (see `lib.rs`).